packages/ag-charts-community/src/chart/interaction/keyNavManager.ts

```diff
@@ -110,7 +114,6 @@ export class KeyNavManager extends InteractionStateListener<KeyNavEventType, Key
             case 'Substract':
                 return this.dispatch('nav-zoom', -1, event);
             case 'Space':
-            case 'Enter':
```

Comment:
Keep this line as a safe-guard.

Although it's probably redundant, it might not be. The `KeyboardEvent.key` property is localised so is therefore not guaranteed to be `'Enter'` on all keyboard layouts.

Reply:
Fair point - reverted this line.